storage.go

```diff
@@ -76,34 +76,38 @@ func (s *Storage) create(path string, opt pairStorageCreate) (o *Object) {
 func (s *Storage) createAppend(ctx context.Context, path string, opt pairStorageCreateAppend) (o *Object, err error) {
 	rp := s.getAbsPath(path)
 
-	options := make([]oss.Option, 0, 3)
-	options = append(options, oss.ContentLength(0))
-	if opt.HasContentType {
-		options = append(options, oss.ContentType(opt.ContentType))
-	}
-	if opt.HasStorageClass {
-		options = append(options, oss.StorageClass(oss.StorageClassType(opt.StorageClass)))
-	}
-	if opt.HasServerSideEncryption {
-		options = append(options, oss.ServerSideEncryption(opt.ServerSideEncryption))
-	}
-	if opt.HasServerSideDataEncryption {
-		options = append(options, oss.ServerSideDataEncryption(opt.ServerSideDataEncryption))
-	}
-	if opt.HasServerSideEncryptionKeyID {
-		options = append(options, oss.ServerSideEncryptionKeyID(opt.ServerSideEncryptionKeyID))
-	}
-
-	offset, err := s.bucket.AppendObject(rp, nil, 0, options...)
```

Comment:
This API call is removed (so won't create an empty object?). Is this expected?

Reply:
similar question here https://github.com/beyondstorage/go-service-qingstor/pull/71/files#diff-8127f6766ba5110c7a99d07305f11d18295fcb2cb08dddb105ab7d55898742c7L113

Reply:
The original intention of creating an empty object in `CreateAppend` is to set the object system metadata at the first append request. You can refer [here](https://github.com/beyondstorage/go-service-qingstor/pull/40#discussion_r621213243) for more details. 
Actually there's no requirement to create an empty object for `CreateAppend`, so I removed the API call to reduce a request.

Reply:
The discussion says that if we cannot create an empty object, we use metadata. 
You mean that we can simply use metadata? This LGTM, but I'm not sure if there is further consideration.

Reply:
> You mean that we can simply use metadata? This LGTM, but I'm not sure if there is further consideration.

I'm not sure whether we should ensoure that the object can be retrieved from server after `CreateAppend`. Or just create an appendable `Object` for users to call `WriteAppend`?

Reply:
> I'm not sure whether we should ensoure that the object can be retrieved from server after `CreateAppend`. Or just create an appendable `Object` for users to call `WriteAppend`?

I think it's not safe. For example, user call `CreateAppend()` but failed on next `WriteAppend` call. So, it's impossible for him to `Stat()` and resume the append process.

Reply:
Got it. I'll change back to create an empty object in `CreateAppend`.